examples/visualization/evoked_topomap.py

```diff
@@ -121,6 +121,24 @@
                     time_unit='s')
 plt.subplots_adjust(left=0.01, right=0.99, bottom=0.01, top=0.88)
 
+# %%
+# We can also highlight specific channels by adding a mask:
+
+times = (0.09, 0.1, 0.11)
+_times = np.searchsorted(evoked.times, times) - 1
```

Comment:
is the `- 1` necessary? In my test you don't need the `- 1`:
```python
import numpy as np

times = np.arange(-0.25, 0.5, 0.01)
request = (0.1, 0.25, 0.3)
_times = np.searchsorted(times, request)

print(times[_times])
```
but in general I prefer `[(np.abs(evoked.times - t)).argmin() for t in times]` as its more accurate (although it is clearly less readable).

Reply:
Yes, it is necessary to set the `True` mask value at the indices picked by plot_topomap. Without the `-1`, none of the 3 plots have a channel highlighted because the indices set to True on the times axis are off by one.

Reply:
Then it is specific to the time values in the example and may not work with other `.times` (as the ones in my example). It would be better then to find the closes timepoint to the one requrested using:
```python
_times = [(np.abs(evoked.times - t)).argmin() for t in times]
```
(bear in mind that people will likely copy that code and expect it to work on their data, where `.times` might be different)

Reply:
I agree with @mmagnuski.  I included the `- 1` in my original example [here](https://github.com/mne-tools/mne-python/issues/9611#issuecomment-891966296) as a quick-and-dirty solution; using `argmin` is more robust.

Reply:
True, I will update later today.